canonical/canonical.com
# Add Simon Aronsson to the hiring leads bios

## Done
Add Simon Aronsson to the hiring leads bios

## QA
- Check the content matches the copy in [the doc](https://drive.google.com/drive/folders/1wTHNsXz2HRhkThxx7buhW6vLskrorX0-)
